merlin/schema/io/tensorflow_metadata.py

```diff
@@ -35,69 +35,197 @@
 
 
 class TensorflowMetadata:
+    """Reads and writes Merlin schemas as Protobuf and JSON files"""
```

Comment:
minor nitpick: I think we could expand on this - since we're not just reading reading and writing schemas as Protobuf/JSON, but specifically we're formatting as the format from the  https://github.com/tensorflow/metadata project. Maybe we can add an extra sentence with a link here?